addons/mail/models/mail_thread.py

```diff
@@ -620,7 +620,7 @@ def _message_track(self, tracked_fields, initial):
         return changes, tracking_value_ids
 
     @api.multi
-    def message_track(self, tracked_fields, initial_values):
+    def message_track(self, tracked_fields, initial_values, **kwargs):
```

Comment:
verify if possible to only accept author_id, not all kwargs to avoid exposing other fields